src/mgr/MgrStandby.cc

```diff
@@ -205,7 +206,13 @@ void MgrStandby::handle_signal(int signum)
   Mutex::Locker l(lock);
   assert(signum == SIGINT || signum == SIGTERM);
   derr << "*** Got signal " << sig_str(signum) << " ***" << dendl;
-  shutdown();
+
+  // FIXME: we can't reliably shut down the python environment.  Just exit.
```

Comment:
I believe It would good to add short description may be very short, why we cannot shut down python here?
And why immediate _exit()